Fix minor issues with the clockwork cult

Cenrus opened the pull request:
Eminence can no longer nuzzle things because it's a giant glowing ball of light, not a puppy. It also has a fancy Master prefix to really stand out on the hierophant network. Formerly this was Clockbrother.

:cl:
fix: The eminence can no longer nuzzle things
fix: The eminence can no longer select itself
fix: The eminence can no longer pull other mobs
fix: The eminence now teleports to random locations on the station, instead of the core
add: The eminence teleport has a sound effect and a flash of color
tweak: The eminence has a "Master" prefix on the hierophant network
tweak: Tweak ark of the clockwork jusiticiar description
/:cl:

Cenrus committed:
emmmmmmy

park66665 commented on code/modules/antagonists/clock_cult/mobs/eminence.dm:
```diff
@@ -47,6 +47,9 @@
 
+/mob/living/simple_animal/eminence/attack_animal(mob/living/simple_animal/M)
+	return
+
```
`attack_animal` is a defense proc; overriding eminence changes how eminence *reacts to an attack by simple animals*. If you want to prevent `attack_animal` being called by eminence (and therefore the nuzzling at `/mob/living/attack_animal` and `/obj/attack_animal`), you can override `UnarmedAttack`.

Cenrus replied:
:+1:

Cenrus committed:
Change the proc to UnarmedAttack, abscond spell check, eminence cannot select itself

Cenrus committed:
Tweak eminence TP, ark spellcheck and desc

Add SFX so it acts more like the servant teleport, now warps to random event locations instead of the core to prevent cheesing with holy water blessed tiles around the eminence TP location

PowerfulBacon commented on code/modules/antagonists/clock_cult/mobs/eminence.dm:
```diff
@@ -109,17 +112,28 @@
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/G = GLOB.celestial_gateway
+	if(G)
+		user.forceMove(get_turf(G))
+		user.playsound_local(user, 'sound/magic/magic_missile.ogg', 50, TRUE)
```
just use SEND_SOUND

PowerfulBacon commented on code/modules/antagonists/clock_cult/mobs/eminence.dm:
```diff
@@ -109,17 +112,28 @@
-	user.forceMove(SSmapping.get_station_center())
+/obj/effect/proc_holder/spell/targeted/eminence/station/cast(mob/living/user)
+	if(is_reebe(user.z))
+		user.forceMove(get_turf(pick(GLOB.generic_event_spawns)))
```
probably better that they go to the center than some random location

Cenrus replied:
I'm worried about people meta-ing the core turf of the station and blessing tiles around it to prevent the eminence doing anything

PowerfulBacon commented on code/modules/antagonists/clock_cult/mobs/eminence.dm:
```diff
@@ -109,17 +112,28 @@
-/obj/effect/proc_holder/spell/targeted/eminence/station/cast(list/targets, mob/living/user)
-	user.forceMove(SSmapping.get_station_center())
+/obj/effect/proc_holder/spell/targeted/eminence/station/cast(mob/living/user)
+	if(is_reebe(user.z))
```
Check if they are not on the station rather than if they are on Reebe, eminence can leave the z-level

PowerfulBacon commented on code/modules/antagonists/clock_cult/mobs/eminence.dm:
```diff
@@ -109,17 +112,28 @@
+/obj/effect/proc_holder/spell/targeted/eminence/station/cast(mob/living/user)
+	if(is_reebe(user.z))
+		user.forceMove(get_turf(pick(GLOB.generic_event_spawns)))
+		user.playsound_local(user, 'sound/magic/magic_missile.ogg', 50, TRUE)
```
send_sound here too

Cenrus committed:
prevent eminence pulling mobs

Cenrus committed:
requested changes

francinum committed:
Properly refactor clockcult prefixes
